keras_nlp/layers/multi_segment_packer_test.py

```diff
@@ -147,6 +147,52 @@ def test_pad_batched_inputs(self):
             ),
         )
 
+    def test_list_special_tokens(self):
+        seq1 = tf.ragged.constant([["a", "b", "c"], ["a", "b", "c"]])
+        seq2 = tf.ragged.constant([["x", "y", "z"], ["x"]])
+        packer = MultiSegmentPacker(
+            9,
+            start_value="[CLS]",
+            end_value="[SEP]",
+            sep_value=["[SEP]", "[SEP]"],
+            pad_value="[PAD]",
+            truncate="round_robin",
+        )
+        output = packer([seq1, seq2])
+        self.assertAllEqual(
+            output,
+            (
+                [
+                    [
+                        "[CLS]",
```

Comment:
try to come up with a slightly shorter test case that will format the lists to one line

Reply:
Whoops!